recipes/tdameritrade/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "tdameritrade" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 692baf738dc8c5f9250764ac7db101171c5df8dd59e39bae77d2a7ff41a02479
+
+build:
+  number: 0
+  skip: True  # [py<37 or (win and vc<14)]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - ipython >=7.0.1
+    - {{ pin_compatible('numpy') }}
```

Comment:
It doesn't look like you're building a C-extension against numpy, so you should just use `numpy` and the appropriate version constraint rather than the pin.